fiona/_shim1.pxd

```diff
@@ -19,6 +19,8 @@ cdef OGRFieldSubType get_field_subtype(void *fielddefn)
 cdef void set_field_subtype(void *fielddefn, OGRFieldSubType subtype)
 cdef bint check_capability_create_layer(void *cogr_ds)
 cdef void *get_linear_geometry(void *geom)
+cdef const char* osr_get_name(OGRSpatialReferenceH hSrs)
```

Comment:
This function is copied from rasterio. It isn't called in fiona yet, but it will be in a future version as we change our modules to be more like those in rasterio.